option optimize_for = LITE_RUNTIME;

message ShopPlayInfoRequest
{
 required uint32 type = 1;
}
